sql/core/src/test/scala/org/apache/spark/sql/execution/ui/AllExecutionsPageSuite.scala

```diff
@@ -178,3 +177,35 @@ class AllExecutionsPageSuite extends SharedSparkSession with BeforeAndAfter {
   }
 }
 
+class AllExecutionsPageWithInMemoryStoreSuite extends AllExecutionsPageSuite {
+  override protected def createStatusStore: SQLAppStatusStore = {
+    val conf = sparkContext.conf
+    kvstore = new ElementTrackingStore(new InMemoryStore, conf)
+    val listener = new SQLAppStatusListener(conf, kvstore, live = true)
+    new SQLAppStatusStore(kvstore, Some(listener))
+  }
+}
+
+class AllExecutionsPageWithRocksDBBackendSuite extends AllExecutionsPageSuite {
+  // TODO: SPARK-41882 remove this field after RocksDB can automatically cleanup
```

Comment:
Instead of this style, please create a new real JIRA ID and point it by using `IDed TODO` style, `TODO(SPARK-12345)`.

Reply:
Thanks for your review and guidance @dongjoon-hyun , let's wait https://github.com/apache/spark/pull/39226, then I think we can clear the TODO in this pr